mne/__init__.py

```diff
@@ -65,7 +65,9 @@
                            write_source_spaces, setup_source_space,
                            setup_volume_source_space, SourceSpaces,
                            add_source_space_distances, morph_source_spaces,
-                           get_volume_labels_from_aseg)
+                           get_volume_labels_from_aseg,
+                           get_volume_labels_from_aseg_AP,
```

Comment:
please remove get_volume_labels_from_aseg_AP

if you need a change in get_volume_labels_from_aseg it's better than duplicating function